controllers/providers/aws/ssm.go

```diff
@@ -60,8 +65,16 @@ func GetAwsSsmClient(region string, cacheCfg *cache.Config, maxRetries int, coll
 }
 
 func (w *AwsWorker) GetEksLatestAmi(OSFamily string, arch string, kubernetesVersion string) (string, error) {
+	return w.GetEksSSMAmi(OSFamily, arch, kubernetesVersion, LatestIdentifiers[OSFamily])
+}
+
+func (w *AwsWorker) GetEksSSMAmi(OSFamily string, arch string, kubernetesVersion string, ssmId string) (string, error) {
```

Comment:
nit: Name is a bit weird, GetEksSsmAmi vs GetEKSSSMAMI vs GetEksSSMAmi - not sure what is the right golang convention but looks like in other methods we dont upcase the abbreviations.
Not really important 🤣